packages/xchain-polkadot/README.md

```diff
@@ -20,3 +20,18 @@ Following peer dependencies have to be installed into your project. These are no
 ```
 yarn add @polkadot/api
 ```
+
+## Service Providers
+
+This package uses the following service providers:
+
+| Function                    | Service          | Notes                                                                                                       |
+| --------------------------- | ---------------- | ----------------------------------------------------------------------------------------------------------- |
+| Balances                    | Subscan          | https://documenter.getpostman.com/view/1618960/TVCe1oRU?version=latest#38c96f38-0278-4510-a030-a3e6be935ee5 |
```

Comment:
Still Subscan by requesting `https://documenter.getpostman.com/...`?

Reply:
https://documenter.getpostman.com/view is the API reference for `getBalances`